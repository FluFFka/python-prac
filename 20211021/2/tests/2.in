:q q 2*q
q 1
q 2
q '123'
q [2]
q f"123"
:format a b c d f"{a+1}+{b+1}+{c+1}+{d+1}={a+b+c+d+4}"
format 1 2 3 4
q 888888888888888888888888888888888888888888888888888888888888888888888888888888888
:empty ''
empty
empty
empty
empty
empty
empty
:a ''
:a ''
:a ''
:a ''
:a ''
:a ''
format -1 -2 0.2 4444444
a
:a ''
a
:a ''
:a ''
:a ''
:empty wqeqwieoqwieoqwieoqwi pp wqeqwieoqwieoqwieoqwi+pp+'not_empty_anymore'
empty 'mm_' 'yes_'
quit "functions:{}\nstrings:{}"
