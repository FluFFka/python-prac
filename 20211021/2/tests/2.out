2
4
123123
[2, 2]
123123
2+3+4+5=14
1777777777777777777777777777777777777777777777777777777777777777777777777777777776






0+-1+1.2+4444445=4444445.2


mm_yes_not_empty_anymore
functions:5
strings:32
